pkg/platform/kube/platform.go

```diff
@@ -19,6 +19,7 @@ package kube
 import (
 	"bytes"
 	"fmt"
+	"github.com/nuclio/nuclio/pkg/common"
```

Comment:
Should be imported in its group...how did it pass lint 😮? could be related to the change you made to Makefile

Reply:
Indeed related, fixed, nice catch